portfolio/src/main/webapp/script.js

```diff
@@ -12,26 +12,35 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+let user = null;
```

Comment:
Any reason for this to be a global variable? Can it just be scoped to `checkUserAuthentication`?

Reply:
because later on when implementing authentication for comments, I want the comments logic to know if that comment is made by the current logged in user